.travis.yml

```diff
@@ -16,4 +16,5 @@ before_script:
     - composer install
 
 script:
+    - ./vendor/bin/phpstan analyse src --level=1
```

Comment:
Instead of defining these arguments here, you should create a `phpstan.neon.dist` file containing this configuration. That makes it easier for contributors to check the code since they only need to run `vendor/bin/phpstan analyse` locally.

Reply:
I agree with you. ✅